src/pages/K8s.vue

```diff
@@ -397,14 +406,16 @@ export default {
       ipcRenderer.invoke('settings-write',
         { kubernetes: { port: value } });
     },
-    handleUpdateTraefik(value) {
+    async handleUpdateTraefik(value) {
       if (value === this.settings.kubernetes.options.traefik) {
         return;
       }
 
       const confirmationMessage = `Kubernetes will restart after ${ value ? 'enabling' : 'disabling' } Traefik. \n\nDo you want to proceed?`;
 
-      if (!confirm(confirmationMessage)) {
+      const confirm = await this.confirm(confirmationMessage);
+
+      if (!confirm.response) {
```

Comment:
This is the opposite of what you intend, but I would explicitly compare results, and I would be even more explicit:
```
const OK_POSITION = 0;
const CANCEL_POSITION = 1;
...
if (confirm.response === CANCEL_POSITION) {
  return;
}
...
const buttons = new Array(2);
buttons[OK_POSITION] = 'OK';
buttons[CANCEL_POSITION] = 'Cancel';
...
'show-message-box', { ... buttons: buttons ... }
```

Reply:
We explicitly check for `cancelPosition` in the `confirm` method now. I also opted to guard in each method that makes use of `confirm` so that we consistently make use of the same pattern throughout. Let me know what you think.